tests/forcetorque/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(GTestSetup)
+
+# Find Gazebo
+find_package(gz-sim7 REQUIRED)
+set(GZ_SIM_VER ${gz-sim7_VERSION_MAJOR})
+
+# Fetch and configure GTest
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
+enable_testing()
```

Comment:
This only works when called in the root CMakeLists.txt, so I suggest to remove it here and add it there.
```suggestion
```